fedmsg/meta/base.py

```diff
@@ -154,7 +154,9 @@ def handle_msg(self, msg, **config):
             return match.groups()[-1] or ""
 
     def title(self, msg, **config):
-        return '.'.join(msg['topic'].split('.')[3:])
+        if msg['topic'][0].isalpha():
```

Comment:
Cool, what about the `msg['topic'].split('.', 3)[-1]`? Not that it matters more just an idea